source/index.md

```diff
@@ -11,11 +13,58 @@ build Ember apps, focusing on the most widely-used features of Ember.js.
 For exhaustive documentation of every Ember feature and API, see the
 [Ember.js API documentation](http://emberjs.com/api/).
 
-These guides are written in Markdown and are
-[available on GitHub](https://github.com/emberjs/guides/).
-If there is something missing, or you find a typo or
-mistake, please help us by filing an issue or submitting a pull
-request. You can easily jump to Github's editor for any page by
-clicking the pencil icon in the upper right corner.
+## Assumptions
+
+While we try to make the guides as beginner-friendly as we can, we must
+establish a baseline so that the guides can keep focused on Ember.js
+functionality. We will try to link to appropriate documentation whenever
+a concept is introduced.
+
+To make the most out of the guides, you should have a working knowledge of:
+
+* **JavaScript** - the language that Ember.js is built with. You can find some
+documentation at the [Mozilla Developer Network][mdn-js]
+* **Promises** - a core concept to how Ember.js deals with asynchrony. [TDK: need a good link][promises]
+* **ES2015 modules** - you will better understand Ember CLI's project structure and import paths if you are comfortable with [ES6 JavaScript Modules][js-modules]
+* **ES2015 syntax** - Ember CLI comes with Babel.js by default so you can
+take advantage of newer language features such as arrow functions, template
+strings, destructuring, and more. You can check the
+[Babel.js documentation][babeljs] or read [Understanding ECMAScript 6][es6]
+online.
+
+## Reporting a problem
+
+Typos, missing words, code samples with errors, these are all considered
```

Comment:
Nitpicky, but since this is the front page, I think it's worth being nitpicky: technically this sentence contains a [comma splice](https://owl.english.purdue.edu/engagement/2/1/34/) that should be fixed.